cloudstate-operator/go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	go.uber.org/atomic v1.5.1 // indirect
 	go.uber.org/multierr v1.4.0 // indirect
 	go.uber.org/zap v1.13.0 // indirect
-	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
+	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7 // indirect
```

Comment:
Build failing on git diff check on this line:

https://app.circleci.com/pipelines/github/cloudstateio/cloudstate/564/workflows/a1f27613-1b50-40da-9834-1d01b10cf8e5/jobs/1767

Reply:
Yeah I'm not sure what triggers that change to happen, when I run exactly the same commands that the build runs, it doesn't remove this line. I just removed it manually.

Reply:
Same for me. Didn't remove the line. Different go versions?

Reply:
Not sure, changing it manually fixed it.